src/main/java/io/epirus/console/project/kotlin/KotlinTestCLIRunner.java

```diff
@@ -41,7 +41,7 @@
     public void run() {
         try {
             new UnitTestCreator(javaWrapperDir, unitTestOutputDir).generateKotlin();
-            Console.exitSuccess(
+            System.out.println(
```

Comment:
nit: shouldn't we be using a logger here?

Reply:
Ditto here, we should rethink how we display stuff to users in general.